jelly/src/main/java/org/kohsuke/stapler/jelly/JellyClassLoaderTearOff.java

```diff
@@ -47,7 +46,7 @@
     /**
      * See {@link JellyClassTearOff#scripts} for why we use {@link WeakReference} here.
```

Comment:
Seems to be dead?

Reply:
Sure, but this seems to be a preëxisting issue that I am making neither better nor worse in this PR.